GECKOInstaller.m

```diff
@@ -1,14 +1,31 @@
 classdef GECKOInstaller
-% GECKOInstaller
-%   Support for installing and uninstalling
-%   Run GECKOInstaller.install to install (will set up the path in MATLAB)
-%   Run GECKOInstaller.uninstall to clear the path from MATLAB
-%   To install, you first need to cd to the GECKO root.
+    % GECKOInstaller
+    %   Support for installing and uninstalling
+    %   Run GECKOInstaller.install to install (will set up the path in MATLAB)
+    %   Run GECKOInstaller.uninstall to clear the path from MATLAB
+    %   To install, you first need to cd to the GECKO root.
 
     methods (Static)
         function install
             sourceDir = fileparts(which(mfilename));
             paths = GECKOInstaller.GetFilteredSubPaths(sourceDir, GECKOInstaller.FILE_FILTER);
+
+            GECKOInstaller.checkRAVENversion('2.8.2'); % Minimum RAVEN version
```

Comment:
Based on [this comment](https://github.com/SysBioChalmers/GECKO/pull/319#discussion_r1238475970), the minimum required version of RAVEN should be bumped up:
```suggestion
            GECKOInstaller.checkRAVENversion('2.8.3'); % Minimum RAVEN version
```